src/gc/gc.cpp

```diff
@@ -25726,7 +26050,7 @@ void gc_heap::background_mark_phase ()
 #endif //MULTIPLE_HEAPS
         }
 
-        disable_preemptive (true);
+        disable_preemptive (TRUE);
```

Comment:
the disable_preemptive has a "bool" parameter type, so this looks like an unintended change (the same few lines below)

Reply:
will fix.